Dockerfile

```diff
@@ -1,19 +1,11 @@
-ARG PHP_VERSION=8.1
-
-FROM php:${PHP_VERSION}-cli
+FROM php:8.1-cli
 
+# Install extensions
 RUN apt-get update && \
     apt-get install -y autoconf pkg-config libssl-dev git unzip libzip-dev zlib1g-dev && \
     pecl install mongodb && docker-php-ext-enable mongodb && \
     pecl install xdebug && docker-php-ext-enable xdebug && \
     docker-php-ext-install -j$(nproc) zip
 
-COPY --from=composer:2.6.2 /usr/bin/composer /usr/local/bin/composer
-
-ENV COMPOSER_ALLOW_SUPERUSER=1
-
-WORKDIR /code
-
-COPY ./ ./
-
-CMD ["bash", "-c", "composer install && ./vendor/bin/phpunit --testdox"]
```

Comment:
These lines were ineffective. I moved some of them to the `docker-compose` file. However, To run the tests, I added scripts to the `composer` to make it easy for users to run the commands.